src/realm/util/interprocess_condvar.cpp

```diff
@@ -190,8 +241,10 @@ void InterprocessCondVar::init_shared_part(SharedPart& shared_part)
 
 void InterprocessCondVar::release_shared_part()
 {
-#ifdef REALM_CONDVAR_EMULATION
+#ifdef REALM_CONDVAR_EMULATION 
+#ifndef _WIN32
     File::try_remove(m_resource_path);
+#endif
 #else
 #endif
```

Comment:
notify_all(), notify(), wait(), release_shared_part(), init_shared_part() and set_shared_part() follow that structure. 

close() and ~InterprocessCondVar() are the only two that do not.

There can't be an #error there because it will make it fail compiling on android/ios... 

"Simple" is not always better. We have often made mistakes because we forgot to update source code in a place that we had overlooked. With this common structure in every method, we reduce that risk